Jenkinsfile

```diff
@@ -1 +1 @@
-buildPlugin(configurations: buildPlugin.recommendedConfigurations())
+buildPlugin()
```

Comment:
The recommended versions include 2.164.1.  So they fail.  When I move to github-api 1.106, I'll be able to revert to jenkins version 2.150.3 as the minimum and will revert this change.

Reply:
You could also specify the configurations you want to test manually, but I think the default is fine even if you don't revert it in the future.